tlr-ui/src/services/rest.js

```diff
@@ -5,7 +5,7 @@ export default async function getJSON(baseURL, options) {
     if (process.env.REACT_APP_ENV_IS_GITPOD) {
       root = process.env.REACT_APP_GITPOD_SERVER_URL;
     } else if (process.env.NODE_ENV === "development") {
-      root = "http://localhost:8080";
+      root = `http://${process.env.REACT_APP_HOST || "localhost"}:8080`;
```

Comment:
I can't use `localhost`, so I set `REACT_APP_HOST` to my VM IP in a `.env` file in `tlr-ui/`.